src/app/components/navbar.tsx

```diff
@@ -14,7 +14,7 @@ export default function Navbar() {
   };
 
   return (
-    <nav className="relative flex items-center justify-between p-4 z-10">
+    <nav className="fixed top-0 w-full flex items-center justify-between p-4 z-10 bg-white shadow-md">
```

Comment:
Usage of "bg-white" can be changed to "bg-(--background)" for color theming consistency and easy recoloring